lib/embulk/output/bigquery.rb

```diff
@@ -227,14 +228,55 @@ def self.configure(config, schema, task_count)
           task['abort_on_error'] = (task['max_bad_records'] == 0)
         end
 
+        if task['time_partitioning'] && task['range_partitioning']
+          raise ConfigError.new "`time_partitioning` and `range_partitioning` cannot be used at the same time"
+        end
+
         if task['time_partitioning']
           unless task['time_partitioning']['type']
             raise ConfigError.new "`time_partitioning` must have `type` key"
           end
-        elsif Helper.has_partition_decorator?(task['table'])
+        end
+
+        if Helper.has_partition_decorator?(task['table'])
+          if task['range_partitioning']
+            raise ConfigError.new "partition decorator doesn't support with `range_partition`"
```

Comment:
```suggestion
            raise ConfigError.new "Partition decorators(`table_name$20250101`) don't support `range_partition`"
```
partition decorator -> partition decorators
https://cloud.google.com/bigquery/docs/partitioned-tables

Someone may want to know what partition decorators mean. 

Please fix the message if you are good at English.